CHANGELOG.md

```diff
@@ -1,3 +1,11 @@
+# Release 2.13.0 08-07-2020
+
+- Update envoy to 7/6/2020 (#217)
+- Create FilterStats in client_cache (#216)
+- Fix bug in processing HTTP response, unit tests for client cache CHECK (#212)
+- Add api/README.md and api/script/update_version.h (#211)
+- Support api config versioning:  add v6 to api folder name and package name (#210)
```

Comment:
remove line 4 and 6.

Reply:
remove line 6

Reply:
Done. Thanks!

Reply:
Please add back "Update Envoy .." line.  It will be useful to users too

Reply:
Done, Thanks!